components/headers/GatesHeader.vue

```diff
@@ -50,6 +50,14 @@ export default class extends Vue {
 <style lang="scss" scoped>
 @import '~/assets/scss/mixins.scss';
 
+.extra-container {
```

Comment:
This is a good example of how if we would use utility-first classes we could be able of write "margin-right-s" for example and we could reuse this class in the rest of the markup.